backend/src/main/java/com/huellapositiva/domain/service/EmailCommunicationService.java

```diff
@@ -66,29 +68,21 @@ public void sendSubmittedProposalRevision(ProposalRevisionEmail proposalRevision
         } else {
             emailTemplate = templateService.getProposalRevisionWithoutFeedbackTemplate(proposalRevisionEmail);
         }
-        Email email = Email.createFrom(proposalRevisionEmail, emailTemplate, from);
-        emailService.sendEmail(email);
+        EmailMessage emailMessage = EmailMessage.createFrom(from, proposalRevisionEmail.getEmailAddress(), "Revisión de tu convocatoria", emailTemplate);
+        emailService.sendEmail(emailMessage);
     }
 
 
 
-    public void sendRecoveryPasswordEmail(EmailRecoveryPassword emailRecoveryPassword) {
-        try {
-            EmailTemplate emailTemplate = templateService.getRecoveryEmailTemplate(emailRecoveryPassword.getHash());
-            Email email = Email.createFrom(emailRecoveryPassword, emailTemplate, from);
-            emailService.sendEmail(email);
-        } catch (RuntimeException ex) {
-            log.error("Failed to send email:", ex);
-        }
+    public void sendRecoveryPasswordEmail(RecoveryPasswordEmail recoveryPasswordEmail) {
+        EmailTemplate emailTemplate = templateService.getRecoveryEmailTemplate(recoveryPasswordEmail.getHash());
+        EmailMessage emailMessage = EmailMessage.createFrom(from, recoveryPasswordEmail.getEmail(), "Cambio de tu contraseña", emailTemplate);
+        emailService.sendEmail(emailMessage);
     }
 
     public void sendConfirmationPasswordChanged(EmailAddress emailAddress) {
-        try {
-            EmailTemplate emailTemplate = templateService.getConfirmationPasswordChangedTemplate();
-            Email email = Email.createFrom(emailAddress, emailTemplate, from);
-            emailService.sendEmail(email);
-        } catch (RuntimeException ex) {
-            log.error("Failed to send email:", ex);
-        }
+        EmailTemplate emailTemplate = templateService.getConfirmationPasswordChangedTemplate();
+        EmailMessage emailMessage = EmailMessage.createFrom(from, emailAddress.toString(), "Confirmacion de cambio de contraseña", emailTemplate);
+        emailService.sendEmail(emailMessage);
```

Comment:
Sometimes it can be good to sometimes contain the Strings on the same topic in an array